dpnp/dpnp_algo/dpnp_algo.pxd

```diff
@@ -96,10 +95,6 @@ cdef extern from "dpnp_iface_fptr.hpp":
 
     DPNPFuncData get_dpnp_function_ptr(DPNPFuncName name, DPNPFuncType first_type, DPNPFuncType second_type) except +
 
-
-cdef extern from "constants.hpp":
-    void dpnp_python_constants_initialize_c(void * py_none, void * py_nan)
-
 cdef extern from "dpnp_iface.hpp":
```

Comment:
Functions below can be also removed from `dpnp/dpnp_algo/dpnp_algo.pxd` declaration.

Reply:
Which ones exactly?
I've checked them, but they seems to be in use.

Reply:
It seems `dpnp_memory_alloc_c`, `dpnp_memory_free_c`, `dpnp_memory_memcpy_c` and `dpnp_rng_srand_c` are not used anywhere in code of the cython extension.
So I proposed to remove from line#103 to line#108.